### ISSUE TITLE
Make it possible to specify reporters by an Array

### ISSUE THREAD

**Issue by munierujp** [YOU54F/cypress-multi-reporters#38]
If `reporterEnabled` of config file is an array, use it as is, without calling `split(',')`.

```json
{
  "reporterEnabled": "spec,json"
}
```

```json
{
  "reporterEnabled": [
    "spec",
    "json"
  ]
}
```

**Comment by YOU54F** [comment]
Hi @munierujp, good idea, please feel free to make a PR 👍

**Comment by YOU54F** [comment]
@brettz9 @munierujp will be released shortly as `1.4.0`
